ui/shared/StyledButton.qml

```diff
@@ -8,6 +8,8 @@ import "../imports"
 Button {
     property string label: "My button"
     property color btnColor: Theme.lightBlue
+    property color btnBorderColor: "transparent"
```

Comment:
Didn't know `transparent` was a valid color. Oh well, I created `Theme.transparent` for nothing then.